history/ReadMe.md

```diff
@@ -0,0 +1,51 @@
+# DBPEDIA HISTORY
+
+DBpedia History allows to extract into a RDF format the history of Wikipedia chapter
+
+
+## Previous work
+
+This DBpedia App is a scala/java version of the first work conducted by the French Chapter : https://github.com/dbpedia/Historic/ 
+
+Fabien Gandon, Raphael Boyer, Olivier Corby, Alexandre Monnin. Wikipedia editing history in DBpedia : extracting and publishing the encyclopedia editing activity as linked data. IEEE/WIC/ACM International Joint Conference on Web Intelligence (WI' 16), Oct 2016, Omaha, United States. <hal-01359575>
+https://hal.inria.fr/hal-01359575
+
+Fabien Gandon, Raphael Boyer, Olivier Corby, Alexandre Monnin. Materializing the editing history of Wikipedia as linked data in DBpedia. ISWC 2016 - 15th International Semantic Web Conference, Oct 2016, Kobe, Japan. <http://iswc2016.semanticweb.org/>. <hal-01359583>
+https://hal.inria.fr/hal-01359583
+
+## A first working prototype
+
+This prototype is not optimized, during its development of it we were faced with the WikiPage type checking constraints that are checked in almost every module of the DBpedia pipeline.
+We hardly copy/paste and renamed all the classes and objects we needed for running the extractors.
+This conception could be easily improved by making WikiPage and WikiPageWithRevision objects inherit from the same abstract object.
+But as a first step, we wanted to touch the less possible DBpedia core module.
```

Comment:
I cannot tell what was intended by `touch the less possible`... This needs a better translation.

Reply:
I changed it by writing "we didn't want to impact the core module", is it better ?